.prettierignore

```diff
@@ -7,7 +7,7 @@ Procfile
 # exclude these directories
 /stumptown/
```

Comment:
Running `yarn prettier-format` prints out that it checks on these files

```
stumptown\content\fr\html\reference\elements\abbr\examples\baliser-une-abréviation\example.html 6ms
stumptown\content\fr\html\reference\elements\abbr\examples\définir-une-abréviation\example.html 12ms
stumptown\content\fr\html\reference\elements\abbr\examples\fournir-une-forme-étendue\example.html 7ms
stumptown\content\fr\html\reference\elements\abbr\examples\mettre-en-forme-une-abréviation\example.html 5ms
```

but not for other files under `stumptown` folder, so I guess it may be some bug with the glob matcher not able to match Unicode folders in Windows?

(related to #260)